CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contributing to KoLmafia
+
+First off, thank you for contributing your time and knowledge. This project
+would not exist if not for volunteers like you!
+
+## Workflow Overview
+
+Some suggested reading (lifted from [Github docs](https://github.com/github/docs/blob/main/CONTRIBUTING.md)):
+
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+In the spirit of Github flow, all work must be done in feature branches
+(typically within your fork of the project), and reviewed by at least one of the
+core developers through Github's Pull Request (PR) feature.
+
+We strongly encourage that you provide tests as part of your pull request, as
+this helps us understand the problem at hand and avoid breaking your change in
+the future.
+
+## Guidelines (adapted from [our previous guide](https://kolmafia.us/threads/25364))
+
+Before creating a PR, we strongly recommend creating a bug in the [KoLmafia
+forums](https://kolmafia.us/forums/bug-reports.24/), in order to provide broader
+context regarding the change, and to discuss whether the change is a good
+idea. We'd hate for you to waste effort and be discouraged from submitting
+future PRs because some fundamental concerns arise during code review.
+
```

Comment:
Can we link to the “how to write a good bug report?” In this paragraph?

Reply:
Do we want to link to that here?  WE aren't reporting bugs on GitHub and reports at KoLmafia are driving activities.